app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,25 +1,231 @@
 @import "govuk/components/accordion/accordion";
 
-.gem-c-accordion {
-  .govuk-accordion__section-heading {
-    // this is a temporary addition to fix the line height when it
-    // is being overridden by styles from govuk-template in collections
-    @include govuk-media-query($until: desktop) {
-      @include govuk-font(24, $weight: bold, $line-height: 1.6);
+$gem-c-accordion-border-width: 3px;
+$gem-c-accordion-bottom-border-width: 2px;
+
+// Buttons within the sections don’t need default styling
+.gem-c-accordion__section-button {
+  @include govuk-font($size: 24, $weight: bold);
+  display: inline-block;
+  margin-bottom: 0;
+  padding-top: govuk-spacing(3);
+}
+
+.gem-c-accordion__section-header {
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+}
+
+.js-enabled {
+  .gem-c-accordion {
+
+    border-bottom: $gem-c-accordion-bottom-border-width solid $govuk-border-colour;
+  }
+
+  .gem-c-accordion__controls {
+    text-align: left;
+  }
+
+  .gem-c-accordion__open-all {
+
+    position: relative;
+    z-index: 1;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: none;
+    cursor: pointer;
```

Comment:
Is this necessary as the assumption is that this class will always only be applied to a button? There might be some browser implications which I'm not aware of.

Reply:
Ah so this was ported over and is inline with other button patterns on govuk, but! this might be a bigger topic of _should_ a button switch to a hand cursor as it's not strictly a link.

Ref [Link 1](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b), [Link 2](https://ux.stackexchange.com/questions/105024/why-dont-button-html-elements-have-a-css-cursor-pointer-by-default), [Link 3](https://twitter.com/feather/status/559756555441434624?s=20)

I've left this for the moment, I'd rather they alll be the same rather than only these button behave different but it might be one worth raising as an issue and discussing wider